modules/ROOT/pages/push/binding.adoc

```diff
@@ -1,8 +1,36 @@
 include::{partialsdir}/attributes.adoc[]
 :service-name: Push Notifications
 
-= Binding a {mobile-client} with the {service-name} service
+== Prerequisites
+
+For *iOS*: you need an Apple Developer account and a provisioning certificate. Have a look at link:apple-setup.html[APNs Setup] for further instructions.
+
+For *Android*: A Google account and access to the Firebase console. See link:google-setup.html[Firebase Setup] to get started.
+
+== Binding a {mobile-client} with the {service-name} service
 
 include::{partialsdir}/generic-binding.adoc[]
 
+When you bind a mobile client to {service-name} a variant is created to store the credentials required to communicate with a push network.
+Those credentials need to be provided in the binding dialog. First pick your *Mobile Client Type*. Depending on the selection you then have to fill out the fields for either Android or iOS.
+
+== Android Clients
+
+You need to provide a *Server Key* and a *Sender ID*. Both can be found in the Firebase console under *Project Settings / Cloud Messaging*. Make sure that you have selected `Android` in the `Mobile Client Type` dropdown.
```

Comment:
I think the preferred way is using `>`:
*Project Settings / Cloud Messaging* -> *Project Settings > Cloud Messaging*